src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/daemon-list/daemon-list.component.ts

```diff
@@ -48,10 +47,30 @@ export class DaemonListComponent implements OnInit, OnDestroy {
 
     this.subs = this.rbdMirroringService.subscribeSummary((data) => {
       this.data = data.content_data.daemons;
+      this.daemonData = this.daemonCount(this.data);
       this.tableStatus = new TableStatusViewCache(data.status);
     });
   }
-
+  daemonCount(data: any) {
```

Comment:
Adding return type hints would be ideal

```suggestion
  daemonCount(data: any) -> TypeReturn {
```

Reply:
Done!!